### PR TITLE
Add rofi_highlight

### PR DESCRIPTION
**Pull request by He-Ro**
This adds the ability to use rofi's highlighting capabilities, that
were introduced in rofi 0.15.4. It has to be enabled by the user.
So existing users, do not see a change in behaviour. If it is enabled,
the active connections are no longer marked with '**' in the beginning,
but highlighted by rofi (in a color depending on the user's
configuration).

In this example, the VPN 'DE - Frankfurt' is active, so it is highlighted by rofi:
nm1

I am not quite sure, which branch I should base this pull request on, master or develop?

### CODE REVIEW

File: networkmanager_dmenu [hunk 1]
```diff
@@ -287,30 +290,43 @@ def get_selection(client, aps, vpns, gsms, others):
...
 
     if not sel.rstrip():
         sys.exit()
 
-    action = [i for i in aps + vpns + gsms + others
-              if str(i) == sel.rstrip("\n")]
-    assert len(action) == 1, "Selection was ambiguous: {}".format(str(sel))
+    if not rofi_highlight:
+        action = [i for i in aps + vpns + gsms + others
+                  if prepend_marker_to_active_action(i) == str(sel.rstrip("\n"))]
```
Comment (firecat53) [inline networkmanager_dmenu]: The variable `prepend_marker_to_active_action` doesn't seem to exist anywhere else and the script will fail if rofi_highlight is defined as `False`.

**Comment by firecat53** [comment]
Also, can you please try and keep line length under 80 char? I know, I know, it's the age of wide-screen monitors :smile: Humor me! If it doesn't make sense to split something (like lines 445 and 446), then don't.

Thanks!

**Comment by He-Ro** [comment]
Addressed your comments. Also added a pylint configuration. If you do not like that, then just do not use the last commit.

**Comment by firecat53** [comment]
I've been using pyflakes as part of python-mode for years without really thinking much about it. Now I see you've used pylint, which has some different checks. And then I see python-mode doesn't use up to date linters. And then I find syntastic. And then I see all the other errors pylint points out. Sigh. Gimme a few days to sort myself out :smile:

**Comment by He-Ro** [comment]
Yes, I also just found syntastic ;) no problem. It is not running away.

**Comment by firecat53** [comment]
Merged!

**Pull request closed without merging**